views/components/Graphs/BarGraphTooltip.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { siteTooltipContent } from '../../fe-utils/siteTooltipContent'
+
+const BarGraphTooltip = ({ classes, active, payload, label }) => {
+  if (active && payload && payload.length) {
+    return (
+      <div className={classes.tooltipContainer}>
+        <div className={classes.tooltipHeaderRow}>
+          <div className={classes.tooltipLabelColumn}>{label}</div>
+          <div className={classes.tooltipValueColumn}>Value / Target(%)</div>
+        </div>
+        {siteTooltipContent(payload).map(({ labelColumn, valueColumn }) => {
+          return (
+            <div
+              className={classes.tooltipContentRow}
+              style={{
+                borderTop: labelColumn === 'Total' ? 'solid gray 1px' : '',
+              }}
+            >
+              <div className={classes.tooltipLabelColumn}>{labelColumn}</div>
+              <div className={classes.tooltipValueColumn}>{valueColumn}</div>
+            </div>
+          )
+        })}
+      </div>
+    )
+  }
+
+  return null
```

Comment:
I think it's common to return null first if conditions are not met and otherwise return the component content. So in this case it would be something like...

```jsx
const BarGraphTooltip = ({ classes, active, payload, label }) => {
  if (!active || !payload?.length) {
    return null
  }

  return (
    ...
  )
}
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/191/commits/82a8267770a707eedb1efe2f1cc9193d667b1d12